recipes/scipy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version="0.17.0" %}
+
+package:
+  name: scipy
+  version: {{ version }}
+
+source:
+  fn: scipy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scipy/scipy-0.17.0.tar.gz
+  md5: 5ff2971e1ce90e762c59d2cd84837224
+
+build:
+  number: 0
+  features:
+    - openblas
+
+requirements:
+  build:
+    - python
+    - numpy     x.x
+    - openblas
+  run:
+    - python
+    - numpy     x.x
+    - openblas
+    - libgfortran 3.0*
```

Comment:
I'm not sure I like this. I would rather we use the `gcc` package for now to build Fortran and OpenMP code and add `libgcc` as a runtime dependency as that is what we have [decided]( https://github.com/pelson/Obvious-CI/pull/61#issuecomment-215355681 ) to do for present. We have not included a Fortran compiler in the docker image because we have not settled this point.